docs/spfx/viva/get-started/actions/geolocation/GeolocationDocumentation.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Documentation for Geolocation action in SharePoint Adaptive Card Extension
+description: Geolocation is a new action that the SharePoint Adaptive Card Extension framework supports, which enables third party developers to come up with their location specific scenarios.
+ms.date: 03/04/2022
+ms.prod: sharepoint
+ms.localizationpriority: high
+---
+# Location capabilities in Adaptive Card Extension
+
+> [!NOTE]
+> This tutorial assumes that you have installed the SPFx v1.14 and have already built your first SharePoint Adaptive Card Extension.<br/>
+> For more information on installing the SPFx v1.14, see [SharePoint Framework v1.14 release notes](../../../../release-1.14.md).
+> To learn how to create your first SharePoint Adaptive Card Extension, try out [this tutorial](../../../get-started/build-first-sharepoint-adaptive-card-extension.md).
+
+### New action types for geolocation
+
+There are 2 Location actions:
+1. Get Location
+2. Show Location
+
+### Get Location:
+Gives user’s current device location or opens a location picker and returns the location chosen by the user. In the browser it uses Bing Maps as the mapping interface.
+
+The ACE action for Get Location is: `VivaAction.GetLocation`. <br/>
+It takes an optional boolean parameter: `ChooseLocationOnMap`.
+
+If the property `ChooseLocationOnMap` is set to `true`, then the action will open a map, and user will get to choose a location on the map, otherwise, it will give user's current device location.
+
+### Show Location:
+With this action, a map shows up on the screen on which you can either show your current location on the map or you can show your specified coordinates on the map.
```

Comment:
```suggestion
With this action, a map shows up on the screen on which you can either show the user's current location on the map or you can show your specified coordinates on the map.
```

Reply:
Added "the user's".
But I am not too happy with "Nothing is returned".
If you are referring to the onAction callback, then that is described in the documentation below.